examples/data-sources/kentik-synthetics_agents/run.sh

```diff
@@ -1,27 +1,46 @@
 #!/usr/bin/env bash
-# Run the example using live Kentik API server.
+# The script applies the example Terraform configuration.
+# The provider uses stub Kentik API server by default.
+# Production Kentik API server can be used by passing "production" positional argument to the script.
 
 SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
 REPO_DIR=$(cd -- "$SCRIPT_DIR" && cd ../../../ && pwd)
 
+PRODUCTION_FLAG="production"
```

Comment:
(non-critical) Using variable for the special value is IMHO counterproductive. I'd put the test right here:

```
...
if [[ $1 == "production" ]]; then
    warn "Warning: Resources in your production Kentik environment may be modified"
    pause
fi
...
```

Reply:
Done